src/main/resources/dota.yml

```diff
@@ -0,0 +1 @@
+key = B946D0A6B0726327F1C9D44095CD11C0
```

Comment:
change it following .yml type structure -> dota.api_key: B946D0A6B0726327F1C9D44095CD11C0

Reply:
Done